backend/docker/dev/Dockerfile

```diff
@@ -55,12 +54,10 @@ RUN /bin/bash -c ". ~/.nvm/nvm.sh && \
 	nvm alias default node && nvm cache clear"
 
 RUN echo export PATH="\
-/root/.nvm/versions/node/${NODE_VERSION}/bin:\
-$(${PYTHON_VERSION} -m site --user-base)/bin:\
-$(python3 -m site --user-base)/bin:\
-$(python -m site --user-base)/bin:\
-/home/${CONTAINER_USER}/.local/bin:\
-$PATH" >> ~/.bashrc && \
+    /root/.nvm/versions/node/${NODE_VERSION}/bin:\
+    $(${PYTHON_VERSION} -m site --user-base)/bin:\
+    $(python3 -m site --user-base)/bin:\
+    $PATH" >> ~/.bashrc && \
```

Comment:
Are these changes part of this PR?

Reply:
Let me remove that. Its not part of this PR. Something I was trying to do during development